### PR TITLE
implemented the hotel scenario

### PR DESCRIPTION
**Pull request by Devorag**
I did the work please review and let me know if anything needs to be changed

### CODE REVIEW

**Comment by Devorag** [comment]
Hatzlocha!!!!!!!!!!!!!!!!!!!!!

File: 0 database.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+use master
+go 
+drop database if exists hotel
```
Comment (YochevedG) [inline 0 database.sql]: use hoteldb instead of just hotel

File: 3 data.sql [hunk 2]
```diff
@@ -0,0 +1,21 @@
+insert guests(LastName, FirstName, HotelLocation, RoomType, FloorNumber, RoomNumber, CheckInDate, CheckOutDate)
```
Comment (YochevedG) [inline 3 data.sql:5]: make sure the data is not added multiple times

Comment (YochevedG) [inline 3 data.sql:5]: make sure this is running directly from the database

File: 4 reports.sql [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
+--I need the following reports:
+--1. show me how much money the glorious hotel chain has made, per location
+SELECT TotalPerLocation = Sum(g.price), g.HotelLocation
```
Comment (YochevedG) [inline 4 reports.sql]: use the total cost, not the price

File: 4 reports.sql [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+SELECT AverageNightlyHotelStays = AVG(g.numdaysstayed), g.HotelLocation
+from guests g 
+group by g.HotelLocation
+--4. show the top 2 customers that paid the most money
+SELECT top(2) *
+from guests g 
+order by g.TotalCostOfStay desc
+--5. for govt purposes, we need a column in this format:
+--first name last name: amount of days stayed, total price paid (Hotel Location)
+select ConcatFormat = concat(g.firstname, '', g.lastname, ': ', g.numdaysstayed, ', ', g.TotalCostOfStay, ' (', g.hotelLocation, ')')
```
Comment (YochevedG) [inline 4 reports.sql]: make sure there is space btwn first and last name

**Review by YochevedG — CHANGES_REQUESTED** [review]

**Comment by YochevedG** [comment]
Awesome job!!!
please see comments and fix
thanx

File: 0 database.sql (hunk 1 above)
Comment (Devorag) [reply 0 database.sql]: added commit to fix

Comment (Devorag) [reply 0 database.sql]: added commit to fix

File: 3 data.sql (hunk 2 above)
Comment (Devorag) [reply 3 data.sql:5]: fixed problem

File: 4 reports.sql (hunk 3 above)
Comment (Devorag) [reply 4 reports.sql]: fixed it!

File: 4 reports.sql (hunk 4 above)
Comment (Devorag) [reply 4 reports.sql]: fixed the issue

**Comment by Devorag** [comment]
Thanks for your feedback and thorough review!!!!!!!! You are doing great!!!!!!!!!!!

File: 0 database.sql (hunk 1 above)
Comment (YochevedG) [reply 0 database.sql]: Great job!

File: 3 data.sql (hunk 2 above)
Comment (YochevedG) [reply 3 data.sql:5]: amazing!

File: 4 reports.sql (hunk 3 above)
Comment (YochevedG) [reply 4 reports.sql]: Perfect:)

File: 4 reports.sql (hunk 4 above)
Comment (YochevedG) [reply 4 reports.sql]: Awesome!

**Review by YochevedG — APPROVED** [review]

**Pull request closed without merging**